CoreFindView/src/au/gov/asd/tac/constellation/views/find2/utilities/ActiveFindResultsList.java

```diff
@@ -102,4 +105,23 @@ public static void addToAdvancedFindResultsList(final FindResultsList additions)
             }
         });
     }
+
+    /**
+     * Gets how many unique graphs are found in the list - will equal the number of graphs open when the search was done
+     *
+     * @param findResultsList
+     * @return
+     */
+    public static int getUniqueGraphCount(final FindResultsList findResultsList) {
+        List<String> uniqueGraphs = new ArrayList();
+        if (findResultsList != null) {
+            for (final FindResult result : findResultsList) {
+                String graphId = result.getGraphId();
+                if (!uniqueGraphs.contains(graphId)) {
+                    uniqueGraphs.add(graphId);
+                }
```

Comment:
Given you are just returning the list size and you're wanting the elements in the list to be unique, could possibly use a Set here instead (would save doing the contains check)